bot/typings/rure.pyi

```diff
@@ -0,0 +1,12 @@
+from re import Match, RegexFlag
+from typing import AnyStr, Optional, Pattern, overload
+
+_FlagsType = RegexFlag
```

Comment:
why do you assign here instead of using RegexFlag directly?

Reply:
idk, I just styled this after type shed